Wikipedia/Code/ArticleViewController+Editing.swift

```diff
@@ -559,9 +559,7 @@ extension ArticleViewController: WMFAltTextPreviewDelegate {
                        navigationController.viewControllers.count > 2 {
                         // pop back two view controllers.
                         let index = (navigationController.viewControllers.count-1) - 2
-                        if let _ = navigationController.viewControllers[index] as? ArticleViewController {
-                            navigationController.popToViewController(navigationController.viewControllers[index], animated: true)
-                        }
+                        navigationController.popToViewController(navigationController.viewControllers[index], animated: true)
```

Comment:
Actually we do want to make sure this view controller is an ArticleViewController before popping. What did the SwiftLint error say?

Reply:
Unused Optional Binding Violation: Prefer `!= nil` over `let _ =` (unused_optional_binding)

I could do something like 

`if navigationController.viewControllers[index] is ArticleViewController {
    navigationController.popToViewController(navigationController.viewControllers[index], animated: true)
}
`

Reply:
That sounds good!